frontend-v2/src/features/main/Sidebar.tsx

```diff
@@ -368,8 +464,13 @@ export default function Sidebar() {
       </AppBar>
       <Box
         component="nav"
-        sx={{ width: { sm: drawerWidth }, flexShrink: { sm: 0 } }}
-        aria-label="mailbox folders"
+        sx={{
+          width: {
+            sm: isExpanded ? drawerExpandedWidth : drawerCollapsedWidth,
+          },
+          flexShrink: { sm: 0 },
+        }}
+        aria-label="navigation"
```

Comment:
The ARIA role of a HTML `<nav>` is 'navigation'. Screen-readers announce the role and label so this would be something like 'navigation navigation'. I'd recommend using `aria-label` to give this `<nav>` a more specific label.
```suggestion
        aria-label="project navigation"
```